MobileBuy/sample/src/main/java/com/shopify/sample/util/UseCase.java

```diff
@@ -0,0 +1,29 @@
+package com.shopify.sample.util;
```

Comment:
why it's in util? should it be under `com.shopify.sample.domain`

Reply:
I picture that all the core / base classes should be in one package. It is either `com.shopify.sample.core` or `com.shopify.sample.base`. So, domain is just the implementation of those core classes (like using a library).

Reply:
yeah lets move them to `com.shopify.sample.core`